### ISSUE TITLE
How to list existing ajaxSniffers?

### ISSUE THREAD

**Issue by marcogomex** [medialab/artoo#303]
Hi,

How can I list existing sniffers?

Best regards,

MG

**Comment by Yomguithereal** [comment]
@marcogomex try looking in `artoo.ajaxSniffer.listeners`.

**Comment by marcogomex** [comment]
Thanks,

Now I can´t turn off the spider.

![off](https://user-images.githubusercontent.com/3374809/148087599-a5aff084-0836-44d4-851b-a2b11da2b477.png)

**Comment by Yomguithereal** [comment]
Yes it seems to be a bug because the actual callback wrapper I bind into my internal listeners is not the same reference as the one you pass sorry. I can try to find some time to fix this.
